dev/io.openliberty.cdi.4.0.internal_fat/fat/src/io/openliberty/cdi40/internal/fat/startupEvents/sharedLib/TestLoggingObserver.java

```diff
@@ -0,0 +1,47 @@
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.cdi40.internal.fat.startupEvents.sharedLib;
+
+import jakarta.annotation.Priority;
+import jakarta.interceptor.AroundInvoke;
+import jakarta.interceptor.Interceptor;
+import jakarta.interceptor.InvocationContext;
+
+@LoggedTest
+@Interceptor
+@Priority(1)
+public class TestLoggingObserver {
+
+    @AroundInvoke
+    public Object runlogTestMethod(InvocationContext invocationContext) {
+
+        String name = "" + invocationContext.getTarget().getClass().getName().replaceAll("\\$Proxy\\$_\\$\\$_WeldSubclass", "") + "." + invocationContext.getMethod().getName();
+        Object returnValue = null;
```

Comment:
Have you run this in full mode - to ensure that this pattern replacement works on all CDI versions?

Reply:
1. This is a CDI 4+ test, so there aren't that many repeats.
2. Yes it ran in full mode in the build.